setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import find_packages
+from setuptools import setup
+
+setup(
+    name="manifest-bouncer",
+    version="0.1.0",
+    license="BSD",
+
+    author="Red Hat App-SRE Team",
+    author_email="sd-app-sre@redhat.com",
+
+    description="Run checks on k8s/openshift manifests",
+
+    packages=find_packages(exclude=('tests',)),
+
+    install_requires=[
+        "anymarkup>=0.8.0,<0.9.0",
+    ],
+
+    test_suite="tests",
+
```

Comment:
Since this is a new project I would suggest we ensure it works on python3 and add `python_requires='>=3.7'`

More and more I find libraries that have dropped support for 2.7 and can't use them in 2.7 codebases such as qontract-reconcile

Reply:
totally! this has been coded in 3.7 and it doesn't support 2.x at all. I thought adding it to the classifiers was enough, I didn't know about python_requires. Thanks! done :)